### PR TITLE
Corrected metadata tag

### PR DESCRIPTION
**Pull request by jdmartinez36**
Corrected all ms.custom metadata tags to ms.custom: seo-marvel-mar2020.

@DaniHalfin

### CODE REVIEW

File: Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ ms.custom:
   - Audio Conferencing
   - seo-marvel-mar2020
 description: An Office 365 admin can edit the Teams Audio Conferencing settings including provider, default toll or toll-free number, conference ID, or PIN for a user.
+description: An Office 365 admin can edit the Teams Audio Conferencing settings, including provider, default toll or toll-free number, conference ID, or PIN for a user.
```
Comment (DaniHalfin) [inline Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md]: @jdmartinez36 - looks like there's a description field already?

File: Teams/FAQ-journey.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ search.appverid: MET150
 f1.keywords:
 - CSH
 ms.custom: Teams-upgrade-guidance
```
Comment (DaniHalfin) [inline Teams/FAQ-journey.md]: @jdmartinez36  - could you please have both ms.custom tags be of the same list format?
as in - 
ms.custom:
  - Teams-upgrade-guidance
  - seo-marvel-mar2020

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by DaniHalfin** [comment]
@scanum - no need to copy edit. I'll assign to myself.
I'll provide an update on our internal channel. Thanks and sorry for the mix up.

File: Teams/FAQ-journey.md (hunk 2 above)
Comment (jdmartinez36) [reply Teams/FAQ-journey.md]: @DaniHalfin - fixed this tag and 7 others in set.

File: Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md (hunk 1 above)
Comment (jdmartinez36) [reply Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md]: @DaniHalfin - I removed the extra description. Sorry, made a mistake while merging changes.

**Pull request merged**